plugins/generator-1.19.4/forge-1.19.4/procedures/entity_get_ticks_required_to_freeze.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity}.getTicksRequiredToFreeze())
```

Comment:
This should have a int marker (`/*@int*/`) at the very beginning (same in 1.20.1)

Reply:
Done